lib/util/devutil.js

```diff
@@ -159,12 +160,17 @@ devutil.makeIdentity = function(serial, properties) {
     model = model.substr(manufacturer.length)
   }
 
-  if (marketName) {
-    var devices = androidDeviceList.getDevicesByDeviceId(marketName)
-    if (devices.length > 0) {
-      marketName = devices[0].name
+  if (isNaN(customMarketName)) {
```

Comment:
NaN should be used only with numeric values.
`customMarketName` is rather a string so we can check whether it is defined. See eg. line 153.

Reply:
ahh good catch 👍  and done 👍